Improve website accessibility

lbeckman314 opened the pull request:
Ticks two of the boxes from #912 

- [x] Include labels for inputs (helps with screen reader accessibility even if visibly hidden)

- [x] Improve keyboard navigation (I suggest that tabbing moves between project titles and skips project tags & "up-for-grabs" label)

I'm still very new to pull requests, so any edits or recommendations on the commits are more than welcome. : )

lbeckman314 committed:
included labels for inputs

lbeckman314 committed:
tabbing moves between project titles

- skips over project tags and "up-for-grabs" labels

allieholcombe commented on _includes/scripts.html:
```diff
@@ -1,15 +1,15 @@
-    <select class="names-filter" multiple data-placeholder="Select a project..." >
+    <label for="project">Filter by name:</label>
+    <select class="names-filter" multiple data-placeholder="Select a project..." id="project">
     <% _.each(names, function(entry, key){ %>
```
Almost identical to what I had in the works, only difference being that my ids were a smidgen more descriptive (`filter-by-name-input` etc.)

lbeckman314 replied:
You probably can't go wrong being more descriptive. In that case I would elect to use your improvements over mine!

ritwik12 replied:
@alexandraholcombe I think you haven't sent a PR, please send a PR.

allieholcombe replied:
@ritwik12 @lbeckman314 This PR includes a few changes mine doesn't include.  I'll put that in this evening, but I recommend merging this first.

shiftkey commented:
@lbeckman314 it looks like there's some conflicts on this PR, and I'm not sure of the resolution. Are you able to take a look?

lbeckman314 committed:
Merge branch 'gh-pages' into improve-website-accessibility

lbeckman314 commented:
Absolutely. I will take a look.

lbeckman314 committed:
Readded accessibility changes.

- Include labels for inputs (helps with screen reader accessibility even if visibly hidden)
- Improve keyboard navigation (I suggest that tabbing moves between project titles and skips project tags & "up-for-grabs" label)

lbeckman314 committed:
Readded accessibility changes.

- Include labels for inputs (helps with screen reader accessibility even if visibly hidden)
- Improve keyboard navigation (I suggest that tabbing moves between project titles and skips project tags & "up-for-grabs" label)

lbeckman314 committed:
Removed unnecessary class for filter labels.

lbeckman314 commented:
Fetched changes from upstream. Resolved and ready for review.

shiftkey reviewed (approved):
LGTM - thanks @lbeckman314!
